packages/pluggableWidgets/video-player-web/package.json

```diff
@@ -31,7 +31,7 @@
     "lint": "..\"/../../node_modules/.bin/eslint\" --config ../../../.eslintrc.js --ext .jsx,.js,.ts,.tsx src/",
     "test": "..\"/../../node_modules/.bin/jest\" --config ../../../scripts/test/jest.web.config.js",
     "test:e2e": "pluggable-widgets-tools test:e2e:web",
-    "test:e2e:dev": "..\"/../../node_modules/.bin/cross-env\" DEBUG=true ../../node_modules/.bin/wdio ../../configs/e2e/wdio.conf.js",
+    "test:e2e:dev": "DEBUG=true URL=http://localhost:8080 pluggable-widgets-tools test:e2e:dev",
```

Comment:
I think it is nicer to make localhost:8080 a default in our wdio config

Reply:
You're right! We already have that in our wdio config.